mandatory/3-playing-computer.js

```diff
@@ -7,12 +7,35 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+     
+      The error is at console.log(b), this is because 'b' is not defined and the error message will log out 'undefined'.
+ 
   2. Remove the line that throws the error.
   3. What is printed to the console?
+      
+      2;
+      6;
+      4;
+      6;
+      8;
+      8;
+      10;
+      
   4. How many times is "f1" called?
+      
+      Two times, when 'i = 1' and 'i = 3'
+
   5. How many times is "f2" called?
+
+      Three times, when 'i = 0'; 'i = 2'; 'i = 4'.
+  
   6. What value does the "a" parameter take in the first "f1" call?
+  
```

Comment:
uh-oh, something has gone wrong here. Try going through this again to see what might have gone wrong.

Reply:
@louisechow , my new value is 8